helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,43 @@
+---
+# Per environment values which override defaults in create-and-vary-a-licence/values.yaml
+
+generic-service:
+  replicaCount: 2
+
+  ingress:
+    host: create-and-vary-a-licence-preprod.hmpps.service.justice.gov.uk
+    contextColour: green
+
+  env:
+    INGRESS_URL: "https://create-and-vary-a-licence-preprod.hmpps.service.justice.gov.uk"
+    HMPPS_AUTH_URL: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
+    TOKEN_VERIFICATION_API_URL: "https://token-verification-api-preprod.prison.service.justice.gov.uk"
+    COMMUNITY_API_URL: "https://community-api.pre-prod.delius.probation.hmpps.dsd.io"
+    PRISON_API_URL: "https://api-preprod.prison.service.justice.gov.uk"
+    LICENCE_API_URL: "https://create-and-vary-a-licence-api-preprod.hmpps.service.justice.gov.uk"
+    PRISONER_SEARCH_API_URL: "https://prisoner-offender-search-preprod.prison.service.justice.gov.uk"
+    PROBATION_SEARCH_API_URL: "https://probation-offender-search-preprod.hmpps.service.justice.gov.uk"
+    PRISON_REGISTER_API_URL: "https://prison-register-preprod.hmpps.service.justice.gov.uk"
+    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    PHASE_BANNER_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    LICENCE_WATERMARK: "true"
+    # A reference to the Gotenberg service - to submit HTML to PDF conversions (internal within namespace)
+    GOTENBERG_API_URL: "http://create-and-vary-a-licence-gotenberg"
+    # A reference to the CVL service - for links in HTML to get images/stylesheets (internal within namespace)
+    LICENCES_URL: "http://create-and-vary-a-licence"
+
+gotenberg:
+  replicaCount: 1
```

Comment:
Only one replica? As preprod is supposed to be the same as prod, is this enough?

Reply:
Again its the balance between being live-like and being wasteful on resources. Its something we can alter quickly if we do a performance test on preprod, but most of the time it will sit idle. Not sure.